CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 # TiDB-Binlog
 All notable changes to this project are documented in this file.
 
+## [2.1.10]
+- Fix all issue reported by static linters of golangci-lint [#599](https://github.com/pingcap/tidb-binlog/pull/599)
+- Drainer: Ignore empty string when getting compressor config [#598](https://github.com/pingcap/tidb-binlog/pull/598)
+- Update parser dependency [#598](https://github.com/pingcap/tidb-binlog/pull/598)
```

Comment:
Shall we document the specific version change here?

Reply:
updated